core/controllers/base.py

```diff
@@ -295,8 +299,71 @@ def dispatch(self):
                 self.handle_exception(e, self.app.debug)
                 return
 
+        try:
+            self.validate_args_schema()
+        except Exception as e:
```

Comment:
Should you catch a narrower exception (or at least handle differently the cases of "not implemented" and "legitimate schema validation error")?

Reply:
Done